testsuite/docs/cucumber-steps.md

```diff
@@ -613,6 +625,44 @@ For example:
   When I call actionchain.add_package_install()
 ```
 
+<a name="b13" />
+
+#### Virtualization
+
+* Create a test virtual machine on a given host
+
+The virtual machine is created without SUSE Manager, directly on the virtual host
+using `qemu-img` and `virt-install`
+
+```cucumber
+  When I create "test-vm" virtual machine on "virt-server"
+```
+
+* Assert a given virtual machine to be in a given state
+
+The possible states are `shut off`, `running` and `paused`.
+
+```cucumber
+  Then I should see "test-vm" virtual machine "shut off" on "virt-server"
+```
+
+* Waiting for a given virtual machine OS to be started
+
+```cucumber
+  When I wait until virtual machine "test-vm" on "virt-server" OS is started
+```
+
+* Check that there no virtual machine of a given name on a host
+
+```cucumber
+  Then I should not see a "test-vm" virtual machine on "virt-server"
+```
```

Comment:
Group with preceding title, and shorten that title?

* Starting a virtual machine
```cucumber
   When I wait until virtual machine "test-vm" is started on "virt-server"
   Then I should not see a "test-vm" virtual machine on "virt-server"
```

Reply:
hum.... I don't see the point here. And your example is just not possible: once a VM is started we should see it on the virt server

Reply:
Merged a few sections together